Apps/Sandcastle/gallery/3D Tiles.html

```diff
@@ -75,6 +75,15 @@
                 }
             }
         }
+        tileset.loadProgress.addEventListener(function(numberOfPendingRequests, numberProcessing) {
+            var test = 0;
```

Comment:
Yeah, I have that on the `TODO`. I was trying to get the console to only log once when there was nothing loading.